src/js/component.js

```diff
@@ -95,6 +95,11 @@ class Component {
     if (options.evented !== false) {
       // Make this an evented object and use `el_`, if available, as its event bus
       evented(this, {eventBusKey: this.el_ ? 'el_' : null});
+
+      if (this.player_) {
+        this.handleLanguagechange = this.handleLanguagechange.bind(this);
+        this.on(this.player_, 'languagechange', this.handleLanguagechange);
+      }
```

Comment:
I don't think that this check is necessary. We make sure that a `this.player_` property is always set on lines 57-61. Removing it locally, it works and tests pass.
```suggestion
      this.handleLanguagechange = this.handleLanguagechange.bind(this);
      this.on(this.player_, 'languagechange', this.handleLanguagechange);
```